xwalk/common/common_api.js

```diff
@@ -167,21 +167,23 @@ var Common = function() {
   //     |postMessage| but wraps the unique identifier as the first argument
   //     automatically.
   //
-  // _addMethod(name, has_callback):
+  // _addMethod(name, has_callback, experimental?):
   //     Convenience function for adding methods to an object that have a
   //     correspondent on the native side. Methods names that start with "_" are
   //     define as not enumerable by default. Set |has_callback| to true if the
   //     method expects a callback as the last parameter.
+  //     Set |experimental| to show warning message to console.
   //
-  // _addMethodWithPromise(name, promise, wrapArgs?, wrapReturns?):
+  // _addMethodWithPromise(name, promise, wrapArgs?, wrapReturns?, experimental?):
```

Comment:
I forgot to change the description for PR #31. So please also add <code>wrapErrorResult?</code>
```
_addMethodWithPromise(name, promise, wrapArgs?, wrapReturns?, wrapErrorResult?, experimental?)
``` 
Same to below methods.